src/averageAges.js

```diff
@@ -15,11 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const diedInCentury = (person, centuryToCheck) => {
```

Comment:
```suggestion
  const checkDeathCentury = (person, centuryToCheck) => {
```
название функции обычно глагол.